dist/prototypes.py

```diff
@@ -240,6 +240,20 @@ def prototypes_os():
     for p in ports:
         output(fns[p], tests[p], f"../src/include/extern_{p}.h")
 
+# Newly generated public headers need to be included from wt_internal.h.
+# Add a warning reminding developers to do so.
+def check_wt_internal_includes():
+    with open("../src/include/wt_internal.h", 'r') as file:
+        lines = file.readlines()
+
+        for mod in SELF_CONTAINED_MODULES:
+            include_line = f"#include \"../{mod}/{mod}.h\""
```

Comment:
This message will have to change once we change the CMake code?

Reply:
Yep. I'm still planning to do that in this PR when there's some spare time.

Reply:
Update on this. I have code to make this work, but there's a few TODOs to investigate, and we need to make a change in MongoDB's `mongo/src/third_party/wiredtiger/BUILD.bazel` file. 
Rather than block this PR I'll spin up a separate ticket for the CMake work. I'll re-request your review in a few minutes